src/main/scala/org/broadinstitute/dsde/firecloud/service/NihService.scala

```diff
@@ -103,6 +105,17 @@ trait NihService extends LazyLogging {
     usersList.getLines().toSet
   }
 
+  def syncWhitelistAllUsers(whitelistName: String): Future[PerRequestMessage] = {
+    println(s"foo $nihWhitelists")
+    nihWhitelists.find(_.name.equals(whitelistName)) match {
```

Comment:
does `if (nihWhitelists.exists(_.name == whitelistName)) ... else ...` suffice? (feels the extra `whitelist` variable is a bit unnecessary)

Reply:
nihWhitelists is a set of objects, not just names. We need to find the right object in the set.